docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM elixir:1.6.0
+FROM elixir:1.8.2
```

Comment:
Again, can we just be surgical for this PR ???

Reply:
Same answer to the `.travis.ci.yml#elixir` update.